content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods/at_confirmation.rst

```diff
@@ -0,0 +1,79 @@
+===========================
+At confirmation reservation
+===========================
+
+The *At Confirmation* reservation method reserves products **only** when a sales order (SO) is
+confirmed, **and** if enough stock of the products included in the :abbr:`SO (sales order)` is
+already available.
+
+.. seealso::
+   :doc:`About reservation methods <../reservation_methods>`
+
+Configuration
+=============
+
+To set the reservation method to *At Confirmation*, navigate to :menuselection:`Inventory app -->
+Configuration --> Operations Types`.
+
+In the :guilabel:`General` tab, locate the :guilabel:`Reservation Method` field, and select
+:guilabel:`At Confirmation`.
+
+.. image:: at_confirmation/at-confirmation-operations-type.png
+   :align: center
+   :alt: Reservation method field on delivery order operation type form.
+
+Workflow
+========
+
+To see the *At Confirmation* reservation method in action, create a new :abbr:`SO (sales order)` by
+navigating to :menuselection:`Sales app --> New`.
+
+Add a customer in the :guilabel:`Customer` field, then, in the :guilabel:`Order Lines` tab, click
+:guilabel:`Add a product`, and select a product to add to the quotation from the drop-down menu.
+Finally, in the :guilabel:`Quantity` column, adjust the desired quantity of the product to sell.
+
+Once ready, click :guilabel:`Confirm` to confirm the sales order.
+
+Click the green :guilabel:`📈 (forecast graph)` icon on the product line to reveal the product's
+:guilabel:`Availability` tooltip. This tooltip reveals the reserved number of units for this order,
+which, in this case, is `100 Units`.
+
+.. image:: at_confirmation/at-confirmation-availability-tooltip.png
+   :align: center
+   :alt: Confirmed sales order with product availability tooltip selected.
+
+.. admonition:: Forecasted Report
+
+   To see all the factors that affect product reservation, click the :guilabel:`View Forecast`
+   internal link arrow to view the :guilabel:`Forecasted Report` page.
+
+   The :guilabel:`Forecasted Report` displays forecast information about the product(s) included in
+   the sales order; namely, any live receipts of the product, and any active sales orders, listed in
+   the :guilabel:`Used By` column. See how each order will be fulfilled in the
+   :guilabel:`Replenishment` column.
+
+   Additionally, the :guilabel:`Forecasted` quantity is calculated at the top of the page, by adding
+   the **On Hand** and **Incoming** quantity, and subtracting the **Outgoing** quantity, as shown
+   below:
```

Comment:
```suggestion
   the :guilabel:`On Hand` and :guilabel:`Incoming` quantity, and subtracting the
   :guilabel:`Outgoing` quantity, as shown below:
```
suggesting the use of GUIs here because they appear on the screen/UI. Same reasoning behind your usage of GUIs for 'Forecasted'

Reply:
Good catch. Suggestion implemented ✅